silk/tools/wpan_util.py

```diff
@@ -37,9 +37,8 @@ def verify(condition):
         error_message = 'verify() failed at line {} in "{}"'.format(calling_frame.f_lineno,
                                                                     calling_frame.f_code.co_filename)
         if not _is_in_verify_within:
-            print error_message
-        return False
-    return True
+            print(error_message)
```

Comment:
can we use logger?

Reply:
Sure. Thanks for the suggestion. Have posted updated code using logger. 
Thanks